linux/scripts/reconf.sh

```diff
@@ -52,11 +56,17 @@ for proj in ${autotool_projects}; do
     fi
 done
 
-if [ "${extra_project}" == "keyman-config" ]; then
-    cd keyman-config/keyman_config
-    sed "s/_VERSION_/${newvers}/g" version.py.in > version.py
-fi
-cd $BASEDIR
+for proj in ${extra_projects}; do
+    if [ "${proj}" == "keyboardprocessor" ]; then
+        rm -rf keyboardprocessor
+        meson ../common/engine/keyboardprocessor keyboardprocessor
```

Comment:
CI build failed here
```
./scripts/reconf.sh: line 62: meson: command not found
```

Reply:
agent and make dist failures fixed, now builds